.github/workflows/ci.yml

```diff
@@ -55,6 +59,13 @@ jobs:
       - name: Check EditorConfig conformance
         if: ${{ always() }}
         run: nix develop --command eclint
+
+      # We only need to run on one system because we're mostly checking for a hash mismatch, which
+      # would be present on any system
+      - name: Nix build package
+        if: ${{ always() }}
```

Comment:
here too

Reply:
...and we'll need to add this to make it fix it like magic: https://docs.determinate.systems/guides/automatically-fix-hashes-in-github-actions